lib/Common/ConfigFlagsList.h

```diff
@@ -903,6 +903,9 @@ FLAGPR_REGOVR_EXP(Boolean, ES6, ES6RegExSymbols        , "Enable ES6 RegExp symb
 FLAGPR_REGOVR_EXP(Boolean, ES6, ES6HasInstance         , "Enable ES6 @@hasInstance symbol"                          , DEFAULT_CONFIG_ES6HasInstanceOf)
 FLAGPR           (Boolean, ES6, ES6Verbose             , "Enable ES6 verbose trace"                                 , DEFAULT_CONFIG_ES6Verbose)
 FLAGPR_REGOVR_EXP(Boolean, ES6, ArrayBufferTransfer    , "Enable ArrayBuffer.transfer"                              , DEFAULT_CONFIG_ArrayBufferTransfer)
+
+FLAGPR           (Boolean, ES6, ObjectGetOwnPropertyDescriptors, "Enable Object.getOwnPropertyDescriptors"              , DEFAULT_CONFIG_ObjectGetOwnPropertyDescriptors)
```

Comment:
Agree with ES prefix to easily differentiate ES feature flags at a distance.

I think we can just use ES going forward and not include the version since the features won't always clearly make it into a particular ES publication in the new yearly release train model.